operations/network/gateway_testing.md

```diff
@@ -7,16 +7,32 @@ This page describes how to run a set of tests to determine if the gateways are f
 
 When the nmnlb network is specified, it will use `api-gw-service-nmn.local` as an override for `nmnlb.<system-domain>` in 1.2. You can set `use-api-gw-override: false` in gateway-test-defn.yaml if you would like to disable that override and use `nmnlb.<system-domain>`.
 
-## Running gateway tests from an NCN
+## Running gateway tests on an NCN
 
-The gateway test script can be found in `/usr/share/doc/csm/scripts/operations/gateway-test`. When `gateway-test.py` is run from an NCN, it has access to the admin client secret using `kubectl`. It will use the admin client secret to get the token for accessing the APIs.
+The gateway test scripts can be found in `/usr/share/doc/csm/scripts/operations/gateway-test`. To test the gateways from an NCN you will need to use gateway-test.py.  When `gateway-test.py` is run from an NCN, it has access to the admin client secret using `kubectl`. It will use the admin client secret to get the token for accessing the APIs.
 
-You can run the test by executing the following command. You will need to specify the system domain (for example, `eniac.dev.cray.com`) and the network you would like to use to obtain the token (for example, `nmnlb`, `can`, `cmn`, or `chn`)
+You can run the test by executing the following command. You will need to specify the system domain (for example, `eniac.dev.cray.com`).
 
 ```bash
-ncn# /usr/share/doc/csm/scripts/operations/gateway-test/gateway-test.py eniac.dev.cray.com nmnlb
+ncn# /usr/share/doc/csm/scripts/operations/gateway-test/gateway-test.py eniac.dev.cray.com
 ```
 
+The test will cycle through all of the test networks specified in gateway-test-defn.yaml.
+
+```bash
+test-networks:
+- name: nmnlb
+  gateway: services-gateway
+- name: cmn
+  gateway: services-gateway
+- name: can
+  gateway: customer-user-gateway
+- name: chn
+  gateway: customer-user-gateway
+```
+
+For each network it will attempt to obtain a token from keycloak.  On an NCN, it should be able to get a token from each of those networks.  It will then use that token to attempt to access each of the services defined in gateway-test-defn.yaml on each of the test-networks.   The test will be able to determine whether it should or should not be able to access the service and it will output a PASS or FAIL for each test depending on the actual results.
```

Comment:
```suggestion
For each network, it will attempt to obtain a token from Keycloak. On an NCN, it should be able to get a token from each of those networks. It will then use that token to attempt to access each of the services defined in `gateway-test-defn.yaml` on each of the `test-networks`. The test will be able to determine whether it should or should not be able to access the service and it will output a PASS or FAIL for each test, depending on the actual results.
```